minecraft_servers/varilx/manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "server_name": "varilx",
   "nice_name": "Varilx Network",
-  "direct_ip": "varilx.de",
+  "direct_ip": "Varilx.DE",
   "server_wildcards": [
     "%.varilx.de"
```

Comment:
Not a valid wildcard, probably overseen in first PR
https://i.imgur.com/0D97BDI.png